src/schema/posts.ts

```diff
@@ -1122,13 +1122,20 @@ export const resolvers: IResolvers<any, Context> = {
       graphorm.query(ctx, info, (builder) => ({
         queryBuilder: builder.queryBuilder
           .innerJoin(
-            (query) =>
-              query
+            (query) => {
+              const sub = query
+                .subQuery()
+                .select('id')
+                .from(PostQuestion, 'pq')
+                .where(`"pq"."postId" = "up"."postId"`);
```

Comment:
Maybe we should add a order by upvotes for the subs, but fully optional, this will do fine

Reply:
There's already an order by votedAt :)